scan.py

```diff
@@ -219,7 +219,7 @@ def lambda_handler(event, context):
         start_scan_time = get_timestamp()
         sns_start_scan(sns_client, s3_object, AV_SCAN_START_SNS_ARN, start_scan_time)
 
-    file_path = get_local_path(s3_object, "/tmp")
+    file_path = get_local_path(s3_object, "/mnt/efs")
```

Comment:
Can we take a look at the other configuration parameters and make a new one for the temporary scan path? Then read that value here? Something like `AV_SCAN_PATH` perhaps? Default the value to "/tmp", then we can simply override with a Lambda Environment variable.

Reply:
Good call, done